Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -593,9 +593,17 @@ script:
             }
             //if query has select and where statements
             else if(args[QUERY] && args[QUERY].toLowerCase().indexOf('select') > -1 && args[QUERY].toLowerCase().indexOf('where') > -1) {
-                args[QUERY] = args[QUERY] + ' && time = "' + buildDateFormat();
+                //if query also have group by statement
```

Comment:
shouldn't it be the other way around with the conditions?

Reply:
I made it look better now, I think that's the cleanest it can get